### PR TITLE
chore: added binary configuration to MPP DSL reference

### PR DESCRIPTION
**Pull request by p7nov**

### CODE REVIEW

File: pages/docs/reference/mpp-dsl-reference.md [hunk 1]
```diff
@@ -270,6 +268,99 @@ kotlin {
 
 </div>
 
+For binaries configuration, the following parameters are available:
+
+|**Name**|**Description**| 
+| --- | --- |
+|`compilation`|The compilation from which the binary is built. By default, `test` binaries are based on the `test` compilation while other binaries - on the `main` compilation.|
+|`linkerOpts`|Options passed to a system linker during binary building.|
```
Comment (katia-energizer) [inline pages/docs/reference/mpp-dsl-reference.md:276]: Is there any list of these options we could link to? Do developers know them?

File: pages/docs/reference/mpp-dsl-reference.md [hunk 2]
```diff
@@ -270,6 +268,99 @@ kotlin {
...
+|`baseName`|Base name for the output file.|
```
Comment (katia-energizer) [inline pages/docs/reference/mpp-dsl-reference.md:277]: Don't you want to add information about the difference between the base and final output file names?

File: pages/docs/reference/mpp-dsl-reference.md [hunk 3]
```diff
@@ -270,6 +268,99 @@ kotlin {
...
+|`entryPoint`|The entry point function for executable binaries. By default, it's `main()` in the root package.|
+|`outpuFile`|Access to the output file.|
```
Comment (katia-energizer) [inline pages/docs/reference/mpp-dsl-reference.md:279]:
```suggestion
|`outputFile`|Access to the output file.|
```
Typo

File: pages/docs/reference/mpp-dsl-reference.md (hunk 1 above)
Comment (p7nov) [reply pages/docs/reference/mpp-dsl-reference.md:276]: Doesn't seem so, system linkers are out of our scope. We just know that they have configuration options. All info is in their help/manuals.

File: pages/docs/reference/mpp-dsl-reference.md [hunk 4]
```diff
@@ -270,6 +268,99 @@ kotlin {
...
+
+|**Name**|**Description**| 
+| --- | --- |
+|`compilation`|The compilation from which the binary is built. By default, `test` binaries are based on the `test` compilation while other binaries - on the `main` compilation.|
+|`linkerOpts`|Options passed to a system linker during binary building.|
+|`baseName`|Custom base name for the output file. The final file name will be formed by adding system-dependent prefix and postfix to this base name.|
+|`entryPoint`|The entry point function for executable binaries. By default, it's `main()` in the root package.|
+|`outputFile`|Access to the output file.|
+|`linkTask`|Access to the link task.|
+|`runTask`|Access to the run task.|
```
Comment (ilmat192) [inline pages/docs/reference/mpp-dsl-reference.md:281]: It is available for executables only. Also this property returns `null` if the binary target is other than `linuxX64`, `macosX64` or `mingwX64`.

**Pull request merged**